src/esp/scene/GibsonSemanticScene.cpp

```diff
@@ -67,7 +79,24 @@ bool SemanticScene::loadGibsonHouse(
       scene.categories_.push_back(category);
       object->category_ = std::move(category);
     }
-    // TODO(msb) add support for aabb
+
+    const auto& jsonCenter = jsonObject["location"];
+    if (!jsonCenter.IsNull()) {
```

Comment:
Code might be simpler if you checked center AND size at the top.

Reply:
I want to create OBB only if at least center is available, while size can be optional.